modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/java/com/liferay/osb/faro/service/impl/FaroProjectLocalServiceImpl.java

```diff
@@ -322,6 +323,10 @@ public FaroProject updateSubscription(
 			return null;
 		}
 
+		if (faroProject.getSubscription() != subscription) {
```

Comment:
@rachaelkoestartyo, I think we should use `subscription.get('name')` instead. `subscription` is a JSON that stores some information that are frequently updated, like `individualsCount` and `pageViewsCount`.